mobile_sam/modeling/image_encoder.py

```diff
@@ -10,7 +10,7 @@
 
 from typing import Optional, Tuple, Type
 
-from .common import LayerNorm2d, MLPBlock
+from common import LayerNorm2d, MLPBlock
```

Comment:
I replaced the relative imports by absolute ones as I was getting errors, happy to revert this back